src/state/network/reducer.js

```diff
@@ -63,6 +64,20 @@ const reducer = (state = initialState, { type, payload }) => {
           .set('data', sortBy(state.get('data'), payload.key, payload.isAcs));
       });
     }
+    case types.ADD_NOTIFICATION: {
+      const existingNotification = state.get('notifications');
+      const lastId = existingNotification.size
+        ? existingNotification.last().id : 0;
+      return state.setIn(['notifications', existingNotification.size], {
```

Comment:
will this show a different alert banner for each error like in our current UI? 
if so, is it possible to combine alerts into one error banner?

Reply:
That's a good suggestion. 💯 

Yes, it's the different alert banner for each `error` same as our current UI, if we want to achieve as you are saying we need to categorize notification store in `info`, `success` and `error`. 
But as of now, we throw only one error when there is an issue in importingHAR file and I don't see any other use of notification for now.
I would say let's come back here once we start dispatching more notifications.